src/app/modules/shared-components/components/add-content/add-content.component.html

```diff
@@ -55,13 +55,13 @@ <h3 i18n>Select a Content</h3>
           <td><span class="type">{{ item.type }}</span></td>
           <td class="main">{{ item.title }}</td>
           <td>
-            <alg-button
-              class="p-small-button p-button-rounded"
-              [icon]="addedIds.includes(item.id) ? 'fa fa-check' : selectExistingText === 'Add' ? 'fa fa-plus' : undefined"
+            <p-button
+              styleClass="alg-button p-small-button p-button-rounded"
+              [icon]="addedIds.includes(item.id) ? 'fa fa-check' : selectExistingText === 'Add' ? 'fa fa-plus' : ''"
               [label]="addedIds.includes(item.id) ? addedText : selectExistingText"
               [disabled]="addedIds.includes(item.id)"
-              (click)="addExisting(item)"
-            ></alg-button>
+              (onClick)="addExisting(item)"
+            ></p-button>
```

Comment:
Vertical alignment of buttons "add" and "already added" is odd, can you check if there's an issue for that and create one if there's none?
![image](https://user-images.githubusercontent.com/16257159/169245624-68c1e63a-089e-4acd-aaec-53203df16e24.png)